### PR TITLE
plugin renaming preparation, SVG data-URLs support and others

### PR DESCRIPTION
**Pull request by AmjadHD**
Image Preview v0.2.0
===========================

## Breaking Changes

- the plugin was renamed to "Image Preview" (previously "Hover Image Preview")
- the settings file was renamed accordingly

## New Features

- added support for SVG data URLs (#29)

## Fixes

- [data-]URL images on long lines where not previewed (#40)

## Notes

- the "max_chars" settings was removed, it was more trouble than it worth
- the default value for "image_folder_name" setting is now "previewed_images" (previously "Hovered Images")

### CODE REVIEW

**Comment by AmjadHD** [comment]
@alvesjtiago, can you review this.

File: main.py [hunk 1]
```diff
@@ -25,60 +29,55 @@
...
+        image_url_re,\
+        image_file_re,\
+        image_file_name_re
+
+    Settings.update(s)
+    # ST popups supported formats
+    ST_FORMATS = {"png", "jpg", "jpeg", "bmp", "gif"}
+    unique_formats_to_convert = set(Settings.formats_to_convert)
+    all_formats = list(ST_FORMATS.union(unique_formats_to_convert))
+    # filter ou ST supported formats
```
Comment (alvesjtiago) [inline main.py]: Missing a `t` on the word out?

File: main.py [hunk 2]
```diff
@@ -25,60 +29,55 @@
...
+    formats_to_convert = tuple('.' + ext for ext in unique_formats_to_convert - ST_FORMATS)
+    # print("[Image Preview]", all_formats, formats_to_convert, unique_formats_to_convert)
```
Comment (alvesjtiago) [inline main.py]: Consider removing commented log.

**Review by alvesjtiago — COMMENTED** [review]
Changes look great, thank you so much @AmjadHD 🙏
Do you ming fixing those two small comments and I'll prepare the migration to the new name on package control.

File: main.py (hunk 1 above)
Comment (AmjadHD) [reply main.py]: Done!

File: main.py (hunk 2 above)
Comment (AmjadHD) [reply main.py]: Done!

**Comment by alvesjtiago** [comment]
@AmjadHD I've just submitted a PR to Package Control (https://github.com/wbond/package_control_channel/pull/7799). As soon as it's merged I'll merge this PR as well. Thank you

**Comment by AmjadHD** [comment]
Hhh, finally.

**Comment by AmjadHD** [comment]
@alvesjtiago, i noticed you didn't make the changes in https://github.com/alvesjtiago/hover-preview/issues/39#issuecomment-549359514

**Comment by alvesjtiago** [comment]
@AmjadHD changing it right now. I will not be changing the repo URL for now. I'll do it in the next version not to disrupt the whole process.

**Comment by alvesjtiago** [comment]
Done - https://github.com/wbond/package_control_channel/pull/7799/commits/b8723a451c025307863410f4a540102ce31f75ba

**Comment by AmjadHD** [comment]
Hi @alvesjtiago, https://github.com/wbond/package_control_channel/pull/7799 has been merged.

**Pull request merged**